gpustack/policies/candidate_selectors/gguf_resource_fit_selector.py

```diff
@@ -534,26 +552,14 @@ def _should_skip_for_allocatable_resource(  # noqa: C901
                 )
                 return True
 
-            else:
-                if not self._model.cpu_offloading:
-                    self._event_collector.add(
-                        EventLevelEnum.ERROR,
-                        EVENT_ACTION_PRE_CHECK,
-                        "Insufficient resources for the model. Please check the available VRAM for the workers.",
-                        reason=EVENT_REASON_INSUFFICIENT_RESOURCES,
-                    )
-                    return True
-                else:
-                    if candidate_func == self.find_single_worker_cpu_candidates:
-                        return False
-
-                    self._event_collector.add(
-                        EventLevelEnum.INFO,
-                        EVENT_ACTION_PRE_CHECK,
-                        "No available resources for workers. Please check workers's available VRAM.",
-                        reason=EVENT_REASON_INSUFFICIENT_RESOURCES,
-                    )
-                    return True
+            if self._gpu_count != 0:
```

Comment:
What's the result when self._gpu_count == 0?

Reply:
Could be cpu worker, will try the find_single_worker_cpu_candidates, if it can't offload to cpu, will have cpu resource related messages.